params/protocol_params.go

```diff
@@ -62,18 +62,22 @@ const (
 
 	// Precompiled contract gas prices
 
-	EcrecoverGas            uint64 = 3000   // Elliptic curve sender recovery gas price
-	Sha256BaseGas           uint64 = 60     // Base price for a SHA256 operation
-	Sha256PerWordGas        uint64 = 12     // Per-word price for a SHA256 operation
-	Ripemd160BaseGas        uint64 = 600    // Base price for a RIPEMD160 operation
-	Ripemd160PerWordGas     uint64 = 120    // Per-word price for a RIPEMD160 operation
-	IdentityBaseGas         uint64 = 15     // Base price for a data copy operation
-	IdentityPerWordGas      uint64 = 3      // Per-work price for a data copy operation
-	ModExpQuadCoeffDiv      uint64 = 20     // Divisor for the quadratic particle of the big int modular exponentiation
-	Bn256AddGas             uint64 = 500    // Gas needed for an elliptic curve addition
-	Bn256ScalarMulGas       uint64 = 40000  // Gas needed for an elliptic curve scalar multiplication
-	Bn256PairingBaseGas     uint64 = 100000 // Base price for an elliptic curve pairing check
-	Bn256PairingPerPointGas uint64 = 80000  // Per-point price for an elliptic curve pairing check
+	EcrecoverGas                     uint64 = 3000   // Elliptic curve sender recovery gas price
+	Sha256BaseGas                    uint64 = 60     // Base price for a SHA256 operation
+	Sha256PerWordGas                 uint64 = 12     // Per-word price for a SHA256 operation
+	Ripemd160BaseGas                 uint64 = 600    // Base price for a RIPEMD160 operation
+	Ripemd160PerWordGas              uint64 = 120    // Per-word price for a RIPEMD160 operation
+	IdentityBaseGas                  uint64 = 15     // Base price for a data copy operation
+	IdentityPerWordGas               uint64 = 3      // Per-work price for a data copy operation
+	ModExpQuadCoeffDiv               uint64 = 20     // Divisor for the quadratic particle of the big int modular exponentiation
+	Bn256AddGas                      uint64 = 500    // Gas needed for an elliptic curve addition
+	Bn256ScalarMulGas                uint64 = 40000  // Gas needed for an elliptic curve scalar multiplication
+	Bn256PairingBaseGas              uint64 = 100000 // Base price for an elliptic curve pairing check
+	Bn256PairingPerPointGas          uint64 = 80000  // Per-point price for an elliptic curve pairing check
+	OptimizedBn256AddGas             uint64 = 50     // Gas needed for an elliptic curve addition using optimized Cloudflare’s bn256 library
```

Comment:
Maybe we put a blank line between the prior precompiles and our new ones, so we also don’t reformat all the old stuff?